examples/robot/robot.ino

```diff
@@ -0,0 +1,302 @@
+//This is an arduino code that can be used to control a robot
+
+//Header inclusions
+#include <LiquidCrystal.h>
+#include <SoftwareSerial.h>
+#include <LedControl.h>
```

Comment:
Please add a comment directing the user where they can get this library. If it's available from Arduino Library Manager, you can use a special link form that opens a Library Manager search in the Arduino IDE:
```c++
#include <LedControl.h>  // http://librarymanager#LedControl
```
but that doesn't work in the Arduino Web Editor, Arduino Pro IDE, or arduino-cli, so it's necessary to add guidance for those users as well. Maybe something like:
```c++
#include <LedControl.h>  // Install the LedControl library via Library Manager. If using the Arduino IDE, click here: http://librarymanager#LedControl
```